legal-api/tests/unit/services/test_document_meta.py

```diff
@@ -16,13 +16,61 @@
 
 Test-Suite to ensure that the Document Meta Service is working as expected.
 """
-
+from legal_api.models import Business
 from legal_api.services import document_meta
-from legal_api.utils.legislation_datetime import LegislationDatetime
 
 
-def test_filing_status_documents_empty(app):
-    """Assert that no documents are returned for non PAID and COMPLETED filing."""
+def factory_business(identifier: str, legal_type: str = ''):
```

Comment:
These methods are already defined in the model tests. Please reuse them

Reply:
Fixed in next commit.